src/CImGui.jl

```diff
@@ -197,6 +197,7 @@ Keyword arguments:
   any tests registered with the test engine they will be queued and run
   automatically.
 - `opengl_version::VersionNumber=v"3.2"`: The OpenGL version to use.
+- `wait_events=false`: Disable calling `WaitEvents()` instead of `PollEvents()`.
```

Comment:
I would reword it to something like:
```suggestion
- `wait_events=false`: Set to `true` to call `GLFW.WaitEvents()` instead of `GLFW.PollEvents()` (only supported for the GLFW/OpenGL backend).
```

I think that's a little clearer.